src/shared/Core/Git.cs

```diff
@@ -219,6 +225,35 @@ public void AddFile(string filePath)
             }
         }
 
+        public void Commit(string message)
+        {
+            EnsureArgument.NotNullOrEmpty(message, nameof(message));
+
+            using (var git = CreateProcess("commit -F -"))
```

Comment:
Why are we using `-F` here? Isn't that for [reading a commit message from a file](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt--Fltfilegt)?

Reply:
Good

Reply:
Love For It